src/App.php

```diff
@@ -518,67 +509,70 @@ function init_pagehead() {
 
 		$tpl = get_markup_template('head.tpl');
 		$this->page['htmlhead'] = replace_macros($tpl, [
-				'$baseurl' => $this->get_baseurl(),
-				'$local_user' => local_user(),
-				'$generator' => 'Friendica' . ' ' . FRIENDICA_VERSION,
-				'$delitem' => t('Delete this item?'),
-				'$showmore' => t('show more'),
-				'$showfewer' => t('show fewer'),
-				'$update_interval' => $interval,
-				'$shortcut_icon' => $shortcut_icon,
-				'$touch_icon' => $touch_icon,
-				'$stylesheet' => $stylesheet,
-				'$infinite_scroll' => $invinite_scroll,
-			]) . $this->page['htmlhead'];
-	}
-
-	function init_page_end() {
+			'$baseurl'         => $this->get_baseurl(),
+			'$local_user'      => local_user(),
+			'$generator'       => 'Friendica' . ' ' . FRIENDICA_VERSION,
+			'$delitem'         => t('Delete this item?'),
+			'$showmore'        => t('show more'),
+			'$showfewer'       => t('show fewer'),
+			'$update_interval' => $interval,
+			'$shortcut_icon'   => $shortcut_icon,
+			'$touch_icon'      => $touch_icon,
+			'$stylesheet'      => $stylesheet,
+			'$infinite_scroll' => $invinite_scroll,
+		]) . $this->page['htmlhead'];
+	}
+
+	public function init_page_end()
+	{
 		if (!isset($this->page['end'])) {
 			$this->page['end'] = '';
 		}
 		$tpl = get_markup_template('end.tpl');
 		$this->page['end'] = replace_macros($tpl, [
-				'$baseurl' => $this->get_baseurl()
-			]) . $this->page['end'];
+			'$baseurl' => $this->get_baseurl()
+		]) . $this->page['end'];
 	}
 
-	function set_curl_code($code) {
+	public function set_curl_code($code)
+	{
 		$this->curl_code = $code;
 	}
 
-	function get_curl_code() {
+	public function get_curl_code()
+	{
 		return $this->curl_code;
 	}
 
-	function set_curl_content_type($content_type) {
+	public function set_curl_content_type($content_type)
+	{
 		$this->curl_content_type = $content_type;
 	}
 
-	function get_curl_content_type() {
+	public function get_curl_content_type()
+	{
 		return $this->curl_content_type;
 	}
 
-	function set_curl_headers($headers) {
+	public function set_curl_headers($headers)
+	{
 		$this->curl_headers = $headers;
 	}
 
-	function get_curl_headers() {
+	public function get_curl_headers()
+	{
 		return $this->curl_headers;
 	}
 
-	function get_cached_avatar_image($avatar_image) {
```

Comment:
This function is in use in two addons.

Reply:
Yeah, I forgot to submit the addons PR.

Reply:
Reminder ;-)

Reply:
Apparently I didn't commit anything yesterday, so this will have to wait tonight.

Reply:
Done.